main/cogs/ladder.py

```diff
@@ -1,14 +1,37 @@
 from discord.ext import commands
 import discord
 import requests
+import copy
 from resources import EnvironmentVariables as ev, ladders
+from resources.characters import Char
 
 modes_body = {
     "communities": 1,
     "active": True
 }
 modes = requests.post("https://api.projectrio.app/tag_set/list", data=modes_body).json()["Tag Sets"]
 
+BASE_GAMES_URL = "https://api.projectrio.app/games/"
+
+# tier_captains should be the same length as tier_min_elo
+tier_captains = [Char.BOWSER, Char.DK, Char.YOSHI, Char.BIRDO, Char.LUIGI, Char.MARIO, Char.PEACH, Char.DAISY, Char.DIDDY, Char.WALUIGI, Char.WARIO]
+tier_min_elo = [2000, 1800, 1600, 1400, 1200, 1000, 800, 600, 400, 200, 0]
+# tier_min_elo 's last number should be 0
```

Comment:
These should probably be defined as a list of tuples, a dict, or some other data structure that directly ties the two lists together. Also think we could have the tiers more densely clustered at ratings that active players commonly achieve (1000-2000) for better progression opportunity in those tiers at the cost of removing a few tiers from the bottom